other/enforce-readwriteonce-pod/.chainsaw-test/chainsaw-step-01-assert-1.yaml

```
@@ -0,0 +1,6 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: enforce-readwriteonce-pod
```

This is not the name of the policy.
```suggestion
  name: readwriteonce-pod
```

---

=== STDERR
        error: resource mapping not found for name: "readwriteonce-pod" namespace: "" from "STDIN": no matches for kind "ClusterPolicy" in version "kyverno.io/v1"
        ensure CRDs are installed first 

this is the error that is coming when I ran the test

---

Kyverno must be installed in a live cluster before running Chainsaw tests.